src/sdltiles.cpp

```diff
@@ -2861,7 +2863,15 @@ static void CheckMessages()
                         break;
 #endif
                     case SDL_WINDOWEVENT_SHOWN:
+                        break;
                     case SDL_WINDOWEVENT_EXPOSED:
+                        break;
```

Comment:
These `break`s here are redundant. Both `SDL_WINDOWEVENT_SHOWN` and `SDL_WINDOWEVENT_EXPOSED` will auto-fallthough to `break` for `SDL_WINDOWEVENT_MINIMIZED`.

Reply:
Mentioned breaks deleted.